libevmasm/RuleList.h

```diff
@@ -662,24 +628,20 @@ std::vector<SimplificationRule<Pattern>> simplificationRuleListPart9(
 	// CREATE
 	rules.push_back({
 		Builtins::AND(Builtins::CREATE(W, X, Y), mask),
-		[=]() -> Pattern { return Builtins::CREATE(W, X, Y); },
-		false
+		[=]() -> Pattern { return Builtins::CREATE(W, X, Y); }
 	});
 	rules.push_back({
 		Builtins::AND(mask, Builtins::CREATE(W, X, Y)),
-		[=]() -> Pattern { return Builtins::CREATE(W, X, Y); },
-		false
+		[=]() -> Pattern { return Builtins::CREATE(W, X, Y); }
 	});
 	// CREATE2
 	rules.push_back({
 		Builtins::AND(Builtins::CREATE2(W, X, Y, Z), mask),
-		[=]() -> Pattern { return Builtins::CREATE2(W, X, Y, Z); },
-		false
+		[=]() -> Pattern { return Builtins::CREATE2(W, X, Y, Z); }
 	});
 	rules.push_back({
 		Builtins::AND(mask, Builtins::CREATE2(W, X, Y, Z)),
-		[=]() -> Pattern { return Builtins::CREATE2(W, X, Y, Z); },
-		false
+		[=]() -> Pattern { return Builtins::CREATE2(W, X, Y, Z); }
```

Comment:
None of these rules will be applied anymore. Maybe we should just remove it?

Reply:
You are right, not even the old optimizer can work with them.